notes/coredns-1.5.0.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "CoreDNS-1.5.0 Release"
+description = "CoreDNS-1.5.0 Release Notes."
+tags = ["Release", "1.5.0", "Notes"]
+release = "1.5.0"
+date = "2019-03-31T09:39:07+00:00"
+author = "coredns"
++++
+
+A new [release](https://github.com/coredns/coredns/releases/tag/v1.4.0): CoreDNS-1.5.0.
+
+**Two** new plugins in this releasse: [*grpc*](/plugins/grpc), and [*ready*](/plugins/ready). And
+some polish and simplications in the core server code.
+
+# Plugins
```

Comment:
Seems we should announce the final deprecation of `TIMEOUT` and `NO_RELOAD` in this release note.